source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateH0443.test.tsx

```diff
@@ -102,6 +106,122 @@ describe('useGenerateH0443 functions', () => {
     });
   });
 
+  it('makes requests to Organization when Property Coordinator is an organization with a Primary Contact', async () => {
+    const organizationPersonMock = getMockPerson({ id: 3, firstName: 'JONH', surname: 'Doe' });
+    getPersonConceptFn.mockResolvedValue(Promise.resolve({ data: organizationPersonMock }));
+
+    const organizationMock = getMockOrganization();
+    getOrganizationConceptFn.mockResolvedValue(Promise.resolve({ data: organizationMock }));
+
+    const responseWithTeam: Api_AcquisitionFile = {
+      ...mockAcquisitionFileResponse(),
+      acquisitionTeam: [
+        {
+          id: 1,
+          acquisitionFileId: 1,
+          teamProfileTypeCode: 'PROPCOORD',
+          rowVersion: 2,
+          organizationId: 100,
+          primaryContactId: 3,
+        },
+      ],
+    };
+    const generate = setup({ acquisitionResponse: responseWithTeam });
+
+    await act(async () => {
+      await generate(0);
+      expect(generateFn).toHaveBeenCalled();
+      expect(getPersonConceptFn).toHaveBeenLastCalledWith(3);
+      expect(getOrganizationConceptFn).toHaveBeenCalledTimes(1);
+    });
```

Comment:
I know this is following an existing pattern in this test file but please consider changing this to:

```tsx
await act(async () => generate(0));
// ...
expect(generateFn).toHaveBeenCalled();
expect(getPersonConceptFn).toHaveBeenLastCalledWith(3);
expect(getOrganizationConceptFn).toHaveBeenCalledTimes(1);
```

The expect statements should be called out of the **act** callback

Reply:
Updated